lib/bedrock.js

```diff
@@ -1,4 +1,4 @@
-/*!
+/*
```

Comment:
Does anyone know why we have `/*!` in the first place?  This says it's a minifier directive: https://stackoverflow.com/a/21832804  Is that applicable in our bedrock backend code?  Was it for some other tool that used that same convention?

Reply:
I have no issues putting this back in if needed.

Reply:
We have this (nearly) everywhere. I don't have strong preferences about it. It does seem like if we don't have a single pattern for this and we *do* want it in frontend files, we'll likely start forgetting to add it there if we remove it here. If we don't care about it for frontend files (we handle marking IP appropriately in some other way) then it's fine to remove.